src/pages/library/DataBrowserToggler.js

```diff
@@ -0,0 +1,52 @@
+import { div, h, strong } from 'react-hyperscript-helpers'
+import { Link, Switch } from 'src/components/common'
+import colors from 'src/libs/colors'
+import { isDataBrowserVisible } from 'src/libs/config'
+import * as Nav from 'src/libs/nav'
+import { useStore } from 'src/libs/react-utils'
+import { authStore } from 'src/libs/state'
+import { catalogPreviewStore } from 'src/pages/library/dataBrowser-utils'
+
+
+export const DataBrowserPreviewToggler = ({ checked }) => {
+  const { user: { id } } = useStore(authStore)
+  catalogPreviewStore.set({ [id]: checked })
+
+  return !isDataBrowserVisible() ? div() : div({
```

Comment:
Should this component be rendered at all if the data browser is not enabled?

Otherwise, could this be simplified to avoid rendering the empty div?

```suggestion
  return !!isDataBrowserVisible() && div({
```

Reply:
I tried doing that initially, and it complained that this function must return an object!